department-of-veterans-affairs/va.gov-cms
# VACMS-16077: Sets Views Pager Heading Default to H2

## Description

Updates patch to set views pager default heading to H2.

Also sets `pagination_heading_level` on content and orphaned_paragraphs view configurations to use default (was H4). 

Relates to #16077 

## Testing done

Performed manual testing and confirmed the following:

- all views pagers rendered w/o issue
- all views that render HTML have the proper `H2` heading

Manual testing was performed on the following URLs:

- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/audit
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/block
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/events
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/outdated_content
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/bulk
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/resources-and-support
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/structure/cm_document
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/cms-content
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/audit/vamc-detail-pages
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/facilities/vamc-prepare-for-visit
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/facilities/addresses
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/facilities
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/facilities/facility-status
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/facilities/vamc-facility-health-services
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/facilities/vamc-facility-non-clinical-services
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/facilities/vamc-system-service-audit
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/facilities/vet-center-services
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/config/development/performance/image-style-warmer/warmup-files
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/audit/knowledge-base-article-administration
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/help/search
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/linky/status
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/linky
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/media/images
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/media
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/messages
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/audit/metatag
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/moderated
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/media/pdfs
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/config/search/redirect
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/redirects-list
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/audit/buttons
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/audit/phone-numbers
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/audit/search
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/audit/tables
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/people
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/user-creation-editing-activity
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/user_history/list
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/promos
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/alerts
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/va-forms/audit
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/va-forms
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/vamc-alerts-and-statuses
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/content/audit/vamc-top-task-page-revision-histories
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/structure/taxonomy/manage/health_care_service_taxonomy/overview
- https://pr16425-3x3dt0nmeldz5nqunbhqdqmdgg87iepp.ci.cms.va.gov/admin/reports/dblog

## Screenshots

![20231214--1034](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/607178/6627f5c3-b78e-4c64-aeb2-87d43c69074d)

## QA steps

What needs to be checked to prove this works?

- Go to any of the links above
- Right-click the pager area and click `Inspect`
- The heading adjacent to the pager's unordered list should be `h2` - similar to the screenshot above

What needs to be checked to prove it didn't break any related things?

The files modified in this commit only affect pagers.


### Definition of Done


### Select Team for PR review
